docs/website/content/2.configuration/4.block/rate_limit.md

```diff
@@ -10,10 +10,9 @@ Rate limiting protects backend services. It implements quota management used to
 | :-------------- | :-------------------- | :--------------- | :--------------- | :--------------- | :--------------- |
 | `period`        | [duration](#duration) | - | Defines the rate limit period. | &#9888; required, must not be negative or `0` (zero) | `period = "1m"` |
 | `per_period`    | integer               | - | Defines the number of allowed backend requests in a period. | &#9888; required, must not be negative or `0` (zero) | `per_period = 100` |
-| `period_window` | string                | `"sliding"` | Defines the window of the period. A `fixed` window always starts with the first request to the parent `backend`. A `sliding` period is a period from the time of the client request that lies behind, e.g. `10:23:44-11:23:45 a.m.` if the request time is `11:23:45 a.m.` and `period = "1h"` is defined. | Allowed values: `"fixed"` or `"sliding"` | `period_window = "sliding"` |
+| `period_window` | string                | `"sliding"` | Defines the window of the period. A `fixed` window permits `per_period` requests within `period` after the first request to the parent backend. After the `period` has expired, another `per_period` requests are permitted. The `sliding` window ensures that only `per_period` requests are sent in any interval of length `period`. | Allowed values: `"fixed"` or `"sliding"` | `period_window = "sliding"` |
```

Comment:
* Does a period "expire"? Perhaps better: "After the `period` has ended"?
* "another ... is permitted" (singular) oder "more ... are permitted" (plural)

Reply:
Done